lib/media/track/sender.js

```diff
@@ -26,7 +30,27 @@ class MediaTrackSender extends MediaTrackTransceiver {
    * @returns {MediaTrackSender}
    */
   clone() {
-    return new MediaTrackSender(this.track.clone());
+    const clone = new MediaTrackSender(this.track.clone());
+    this._clones.add(clone);
+    return clone;
+  }
+
+  /**
+   * Set the given MediaStreamTrack.
+   * @param {MediaStreamTrack} mediaStreamTrack
+   * @returns {Promise<void>}
+   */
+  setMediaStreamTrack(mediaStreamTrack) {
+    const clones = Array.from(this._clones);
+    const senders = Array.from(this._senders);
+    return Promise.all(clones.map(clone => {
+      return clone.setMediaStreamTrack(mediaStreamTrack.clone());
+    }).concat(senders.map(sender => {
+      return sender.replaceTrack(mediaStreamTrack).catch(() => {});
```

Comment:
@makarandp0 
```suggestion
      return sender.replaceTrack(mediaStreamTrack).catch(() => {
        log.warn(...);
      });
```

Reply:
adding log here would need some plumbing, and this error is seen often when peer connection is in closed state.

Reply:
but let me add that warn anyways. It would be useful.